tests/helpers/index.ts

```diff
@@ -0,0 +1,82 @@
+import faker from 'faker';
+
+import { DumpMetadataResponse, IDumpMetadata } from '../../src/dumpMetadata/models/dumpMetadata';
+import { DumpMetadataFilter } from '../../src/dumpMetadata/models/dumpMetadataFilter';
+import { SortFilter } from '../../src/dumpMetadata/models/dumpMetadataFilter';
+
+interface IntegrationDumpMetadataResponese extends Omit<DumpMetadataResponse, 'timestamp'> {
+  timestamp: string;
+}
+
+export const mockObjectStorageConfig = {
+  protocol: 'http',
+  host: 'some_storage_host',
+};
+
+export const DEFAULT_SORT = 'desc';
+
+export const DEFAULT_LIMIT = 10;
+
+export const BOTTOM_FROM = faker.date.past();
+
+export const TOP_TO = faker.date.future(undefined, BOTTOM_FROM);
```

Comment:
you can run the function without params, as future will always be ahead of the past.

Reply:
nice.